samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AuthorMessages.properties

```diff
@@ -152,6 +154,8 @@ cert_rem_assmt=Are you sure you want to remove this assessment titled {0}?
 button_edit_questions=Edit Questions
 button_return_to_main=Return to Main Questions Screen
 cert_rem_q=Are you sure you would like to remove this question?
+cert_rem_submissions1=The assessment 
+cert_rem_submissions2=has submissions. In order to delete this assessment, you must first delete all submissions associated with it.
```

Comment:
Can you combine these two messages into one with a parameter?  Ex.:

cert_rem_submissions=The assessment {0} has submissions. In order to delete......